Szymon/Zadanie7/src/test/java/Szymon7/test/TransferFundsTest.java

```diff
@@ -0,0 +1,41 @@
+package Szymon7.test;
+
+import Szymon7.User;
+import Szymon7.assertion.TransferCompleteAssertion;
+import Szymon7.page.accountServicesPage.AccountServicesPage;
+import Szymon7.page.WelcomePage;
+import Szymon7.scenario.FullRegisterUserScenario;
+import Szymon7.scenario.LoginScenario;
+import Szymon7.test.common.SeleniumTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class TransferFundsTest extends SeleniumTest {
+    private User user = new User();
+
+    @BeforeMethod
+
+    public void eachTestSetup() {
+        user.setUsername();
+        new WelcomePage(driver, "http://parabank.parasoft.com", user)
+            .run(new FullRegisterUserScenario(), user)
+            .run(new LoginScenario(user), user);
+    }
+
+    @Test
+    public void transferFundsTest (){
+        Integer amount = 666;
+        new AccountServicesPage(driver)
+            .clickTransferFundsLink()
+            .enterAmountToTransfer(amount)
+            .clickTransferButtonAndRedirect()
+        .check(new TransferCompleteAssertion())
+            .assertThatConfirmationTextIsPresent();
```

Comment:
Failuje tutaj poniewaz najprawdopodobniej asercja wykonuje sie za szybko i odpowiedni tekst nie zdazyl sie jeszcze pokazac.
Musisz w metodzie asercyjnej poczekac na spodziewany tekst jakis okreslony czas, a jak poleci timeout to wtedy failowac.

Powod, dla ktorego w debugu widzisz poprawny tekst jest taki, ze jak wolasz ```WebElement.getText()``` to on na zywo odczytuje te dane ze strony, gdzie tekst jest juz prawidlowy.